sql/src/test/java/io/crate/testing/SQLTransportExecutor.java

```diff
@@ -347,16 +345,8 @@ private SQLResponse executeWithPg(String stmt,
     }
 
     private static Object toJdbcCompatObject(Connection connection, Object arg) {
-        if (arg == null) {
-            return null;
-        }
-        if (arg instanceof Map) {
-            // setObject with a Map would use hstore. But that only supports text values
```

Comment:
it does not seem to be valid any longer https://github.com/crate/pgjdbc/commit/065206f19b0823f471f7c930476ba4b54f4302b1#diff-4c0a45b2847b54f3cb783aa75ce2f0b8L563